pkg/grillelasticsearch/assertions.go

```diff
@@ -51,3 +52,24 @@ func (ge *ElasticSearch) AssertItemsCount(index string, count int, expected []js
 		return nil
 	})
 }
+
+func (ge *ElasticSearch) AssertScript(name string) grill.Assertion {
```

Comment:
what is the purpose of assertscript ?? 

do we create and remove scripts in runtime, or is it a one time thing at the beginning ?

Reply:
We are creating and deleting also in run time.